src/sentry/runner/commands/configoptions.py

```diff
@@ -0,0 +1,150 @@
+import click
+
+from sentry.runner.decorators import configuration
+
+
+@click.group()
+def configoptions():
+    "Manages Sentry options."
+
+
+@configoptions.command()
+@click.option("--key", "-k", required=True, help="The key of the option to fetch.")
+@configuration
+def fetch(key):
+    "Fetches the option for the given key."
+    click.echo(get(key))
+
+
+@configoptions.command()
+@configuration
+def list():
+    "Fetches all options."
+    from sentry import options
```

Comment:
I think you may need to import settings to be able to import options 
`from django.conf import settings`